frontend/azlinux/azlinux3.go

```diff
@@ -60,6 +60,10 @@ func (w azlinux3) Install(pkgs []string, opts ...installOpt) llb.RunOption {
 	return dalec.WithRunOptions(tdnfInstall(&cfg, "3.0", pkgs), w.tdnfCacheMount(cfg.root))
 }
 
+func (w azlinux3) InstallWithReqs(deps map[string]dalec.PackageConstraints, opts ...installOpt) installFunc {
```

Comment:
This extra function on the interface doesn't seem since the implementations are identical.
To note: I think passing the hardcoded target name below is not correct since that should all be threaded through the request.

Reply:
*doesn't seem necessary